src/Confluent.Kafka/AdminClient.cs

```diff
@@ -1654,51 +1654,51 @@ public Handle Handle
 
         /// <summary>
         ///     Releases all resources used by this AdminClient. In the current
-        ///     implementation, this method may block for up to 100ms. This 
-        ///     will be replaced with a non-blocking version in the future.
+        ///     implementation, this method may block for up to 100ms. It is
```

Comment:
This comment may be counterintuitive, as you’re implementing IAsyncDisposable conditionally (behind a compiler flag).

Reply:
Updated.